chapters/extensions/ray_tracing.adoc

```diff
@@ -226,3 +226,39 @@ the operation:
 After an operation has completed (i.e. `vkDeferredOperationJoinKHR` has returned
 `VK_SUCCESS`), call `vkGetDeferredOperationResultKHR` to get the result of the
 operation.
+
+[[ray-tracing-best-practices]]
+== Ray Tracing Best Practices
+
+=== Minimize the Number of Concurrently Active Ray Query Objects
+
+Ray query objects may be expensive in terms of thread private storage, so for 
+performance, it's best to use as few as possible. In most cases it should be 
+possible to use a single ray query object even if tracing multiple rays since a
+terminated ray that's issuing a new ray can use the same object. Multiple ray
+queries in the same shader should only be needed when multiple traversals need
+to be active concurrently, and shaders should be designed to minimize the number
+of active traversals.
+
+=== Minimize the Size of Ray Payloads, Hit Attributes and Callable Data
+
+The ray tracing shader stages can communicate parameters and results using ray
+payload structures between all traversal stages, ray attribute structures
```

Comment:
as pointed out in https://github.com/KhronosGroup/Vulkan-Guide/pull/209 

should `ray attribute structures` be `hit attribute structures`? @dgkoch @r-potter

Reply:
@spencer-lunarg Yes it should. That's what I get for shamelessly stealing a sentence from the Khronos RT announcement blog post 😀